sql-statements/sql-statement-create-table.md

```diff
@@ -10,49 +10,43 @@ aliases: ['/docs-cn/v2.1/sql-statements/sql-statement-create-table/','/docs-cn/v
 
 ## 语法图
 
-**CreateTableStmt:**
+```ebnf+diagram
+CreateTableStmt ::=
+    'CREATE' OptTemporary 'TABLE' IfNotExists TableName ( TableElementListOpt CreateTableOptionListOpt PartitionOpt DuplicateOpt AsOpt CreateTableSelectOpt | LikeTableWithOrWithoutParen )
 
-![CreateTableStmt](/media/sqlgram/CreateTableStmt.png)
+IfNotExists ::=
+    ('IF' 'NOT' 'EXISTS')?
 
-**IfNotExists:**
+TableName ::=
+    Identifier ('.' Identifier)?
 
-![IfNotExists](/media/sqlgram/IfNotExists.png)
+TableElementListOpt ::=
+    ( '(' TableElementList ')' )?
 
-**TableName:**
+TableElement ::=
+    ColumnDef
+|   Constraint
 
-![TableName](/media/sqlgram/TableName.png)
+PartitionOpt ::=
+    ( 'PARTITION' 'BY' PartitionMethod PartitionNumOpt SubPartitionOpt PartitionDefinitionListOpt )?
 
-**TableElementListOpt:**
+ColumnDef ::=
+    ColumnName ( Type | 'SERIAL' ) ColumnOptionListOpt
 
-![TableElementListOpt](/media/sqlgram/TableElementListOpt.png)
+ColumnName ::=
+    Identifier ( '.' Identifier ( '.' Identifier )? )?
 
-**TableElement:**
+Type ::=
+    NumericType
+|   StringType
+|   DateAndTimeType
 
-![TableElement](/media/sqlgram/TableElement.png)
+ColumnOptionListOpt ::=
+    ColumnOption*
 
-**PartitionOpt:**
-
-![PartitionOpt](/media/sqlgram/PartitionOpt.png)
-
-**ColumnDef:**
-
-![ColumnDef](/media/sqlgram/ColumnDef.png)
-
-**ColumnName:**
-
-![ColumnName](/media/sqlgram/ColumnName.png)
-
-**Type:**
-
-![Type](/media/sqlgram/Type.png)
-
-**ColumnOptionListOpt:**
-
-![ColumnOptionListOpt](/media/sqlgram/ColumnOptionListOpt.png)
-
-**TableOptionListOpt:**
-
-![TableOptionListOpt](/media/sqlgram/TableOptionListOpt.png)
+TableOptionListOpt ::=
+    '' | TableOption* ( ',' TableOption* )*
```

Comment:
@kennytm Please take a look at these lines as well. Thanks!

Reply:
lolwut 🤣 

```suggestion
TableOptionListOpt ::=
    (TableOption (','? TableOption)*)?
```

also `CreateTableOptionListOpt` is equivalent to `TableOptionListOpt`.